client/packages/programs/src/JsonForms/common/components/Header.tsx

```diff
@@ -1,25 +1,31 @@
 import { RegexUtils } from '@common/utils';
 import {
-  LayoutProps,
   uiTypeIs,
   rankWith,
-  UISchemaElement,
+  ControlProps,
 } from '@jsonforms/core';
-import { withJsonFormsLayoutProps } from '@jsonforms/react';
+import { withJsonFormsControlProps } from '@jsonforms/react';
 import { Typography } from '@mui/material';
 import React from 'react';
+import { z } from 'zod';
+import { useZodOptionsValidation } from '../hooks/useZodOptionsValidation';
 
 export const headerTester = rankWith(10, uiTypeIs('Header'));
 
-type HeaderProps = LayoutProps & {
-  uischema: UISchemaElement & {
-    header?: string;
-    subheader?: string;
-  };
-};
+const Options = z.object({
+  header: z.string().optional(),
+  subheader: z.string().optional(),
+}).strict().optional();
```

Comment:
Wonder if at least the header should be required? (and if yes make the whole Options required) Otherwise the control wouldn't show anything... Or at least header or subheader must me set...